clients/client-rds-data/RdsDataServiceClient.ts

```diff
@@ -73,12 +73,25 @@ export type ServiceOutputTypes =
   | ExecuteStatementResponse
   | BatchExecuteStatementResponse;
 
-export interface RDSDataRuntimeDependencies {
+export interface RDSDataSharedConfig {
   /**
    * The function that will be used to populate serializing protocol
    */
-  protocol: string;
-
+  protocol?: string;
+  /**
+   * The service name with which to sign requests.
+   */
+  signingName?: string;
+  /**
+   * The service name with which to construct endpoints.
+   */
+  service?: string;
```

Comment:
This should be removed if we are going to take https://github.com/aws/aws-sdk-js-v3/pull/472